### ISSUE TITLE
Error disabling mod

### ISSUE THREAD

**Issue by Chik3r** [software-inc-multiplayer/software-inc-multiplayer#41]
**Describe the bug**
When deactivating the mod, it gives an error to the output_log.txt (no error on console)

**To Reproduce**
1. Go to main menu
2. Enable mod
3. Disable mod
4. Get pop-up saying `Failed activating mod. Check log for details`
5. Check log (In SINC install dir)

**Expected behaviour**
No error

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Software Inc Version: Alpha 11.6.11
 - Mod Version: v0.3.1-closed
 
**Additional context**
Error:
```
Error activating/deactivating dll mod Multiplayer.Core:
System.TypeInitializationException: The type initializer for 'Multiplayer.Networking.Server' threw an exception. ---> System.NullReferenceException: Object reference not set to an instance of an object
  at MarketSimulation.get_Active () [0x00000] in <9ea9c027dc0040299fd0b91d02ca7bd1>:0 
  at Multiplayer.Networking.GameWorld.Server..ctor () [0x000fa] in <785cc9427bfa4f6f96d6a9e0ad90d482>:0 
  at Multiplayer.Networking.ServerData..ctor (System.String fname) [0x0009b] in <785cc9427bfa4f6f96d6a9e0ad90d482>:0 
  at Multiplayer.Networking.Server..cctor () [0x00035] in <785cc9427bfa4f6f96d6a9e0ad90d482>:0 
   --- End of inner exception stack trace ---
  at ModController+DLLMod.Activate (System.Boolean active) [0x00058] in <9ea9c027dc0040299fd0b91d02ca7bd1>:0 
  at OptionsWindow+<AddModOption>c__AnonStorey9.<>m__0 (System.Boolean x) [0x00000] in <9ea9c027dc0040299fd0b91d02ca7bd1>:0 
```
Only happens in main menu, but it also happens in gameworld if it already threw this error.

**Comment by ghost** [comment]
Hello! I think the problem is in the LaunchBehaviour.cs, its telling the server to stop when the server hasn't even been initalized yet. I'll catchup soon.

**Comment by ghost** [comment]
Ah yes, the IF statement tries to check it before checking if it is null.

Will be fixed in 0.3.2
